toil_orchestrator/tasks.py

```diff
@@ -205,6 +211,8 @@ def check_status_of_jobs(self):
                 except Exception as e:
```

Comment:
I'm not sure what the `job.status == Status.CREATED` is doing -- is this for restarts? I'd like to refactor this at some point because it adds an additional entry of created jobs to here. Jobs can now turn pending here OR `submit_job_to_lsf` which feels troublesome.